lib/project_types/script/layers/application/create_script.rb

```diff
@@ -18,10 +18,19 @@ def call(ctx:, language:, script_name:, extension_point_type:, no_config_ui:)
                 extension_point_type: extension_point_type,
                 language: language
               )
-              project_creator = Infrastructure::Languages::ProjectCreator
-                .for(ctx, language, extension_point, script_name, project.id)
+
+              # to be overwritten with a CLI argument - for now we'll just overwrite it here during dev
+              branch = "add-package-json"
+
+              # remove the need to pass the whole extension-point object to the infra layer
+              repo = extension_point.sdks.for(language).repo
+              type = extension_point.dasherize_type
+              domain = extension_point.domain
+
+              project_creator_input = [ctx, language, domain, type, repo, script_name, project.id, branch]
```

Comment:
I saw we were using `OpenStruct` elsewhere in the code so I updated the code to use this. Is this appropriate?
I don't think OpenStruct actually "defines the contract", so this might have the same problem as the arbitrary array. In any case, it definitely feels cleaner.

Reply:
We went back and forth during pairing and ultimately settled on just passing a hash.

Reply:
I think we should prefer to use keyword arguments (`method(one:, two:)`) instead of a hash. Keyword arguments will not trigger the `Metrics/ParameterLists` cop, if thats what you're worried about.

Reply:
Love this suggestion @jacobsteves . Went ahead and did it that way.